### DISCUSSION TITLE
第3节课作业(4班)

### DISCUSSION THREAD

**Discussion by FlamingoPg** [InternLM/Tutorial#202] [作业]
请大家把第三节课的作业交到这里哦~，并且带上自己的群昵称

**Comment by zhjunqin** [comment]
zhjunqin: https://github.com/zhjunqin/InternLM-tutorial/tree/master/Third

**Comment by helonggood** [comment]
【基于 InternLM 和 LangChain 搭建养猪知识库】
http://t.csdnimg.cn/6w1bD

**Comment by onlyzer0** [comment]
广州cirtwalk：https://blog.csdn.net/qq_42907812/article/details/135512101?spm=1001.2014.3001.5502
onlyzer0

**Comment by phbst** [comment]
https://thankful-archeology-738.notion.site/65d2da54492248c4ad347a5d6d77923d?pvs=4
彭宏博
